init.sh

```diff
@@ -7,6 +10,8 @@ LOGLEVEL="info"
 # to trace evm
 #TRACE="--trace"
 TRACE=""
+# To remove previous installation folders (-> ~/.evmosd*)
+REMOVE_FOLDERS=true
```

Comment:
Should this be set to `REMOVE_FOLDERS=false` initially? Might be safer for everybody who's just cloning the repo and executing `./init.sh` before saving previous configs.

Reply:
yeah, let's use false

Reply:
Obsolete, since the `./init.sh` now does not touch any actual `.evmosd` folders anymore, but instead creates a `.temp-evmosd` folder, which is overwritten when executing the script multiple times